docs/custom-plugins/custom-config.md

```diff
@@ -9,22 +9,24 @@ configuration file.
 The following is an example plugin, defining two configuration bundles:
 
 ```js
-module.exports = {
-  id: 'my-local-plugin'
-  configs: {
-    all: {
-      rules: {
-        'operation-id-not-test': 'error',
-        'boolean-parameter-prefixes': 'error',
+module.exports = function myLocalPlugin() {
```

Comment:
Does this function name need to be a particular value, or where is it used?

Reply:
The function name does not matter. It's not really used anywhere.

Reply:
we can also have anonymous function like:

```
module.exports = function () {
```

BUT the name will appera in the error stacktrace (if error happens) so may help to debug.